packages/gtk3.rb

```diff
@@ -3,24 +3,13 @@
 class Gtk3 < Package
   description 'GTK+ is a multi-platform toolkit for creating graphical user interfaces.'
   homepage 'https://developer.gnome.org/gtk3/3.0/'
-  version '3.24.24-1'
+  @_ver = '3.24.25'
+  version @_ver
   compatibility 'all'
-  source_url 'https://download.gnome.org/sources/gtk+/3.24/gtk+-3.24.24.tar.xz'
-  source_sha256 'cc9d4367c55b724832f6b09ab85481738ea456871f0381768a6a99335a98378a'
-
-  binary_url ({
-     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gtk3-3.24.24-1-chromeos-armv7l.tar.xz',
-      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gtk3-3.24.24-1-chromeos-armv7l.tar.xz',
-        i686: 'https://dl.bintray.com/chromebrew/chromebrew/gtk3-3.24.24-1-chromeos-i686.tar.xz',
-      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gtk3-3.24.24-1-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-     aarch64: '41067ff7cb6dd415c1ffaee6d9016388323a51d7bc7183fe0ef99749bf95dbea',
-      armv7l: '41067ff7cb6dd415c1ffaee6d9016388323a51d7bc7183fe0ef99749bf95dbea',
-        i686: 'e338cd7188bdd23b6675db7013098b8a13cd7fe52e9d3903560037588156f8e8',
-      x86_64: 'c9a7c3e02a3b1f283076e4dfba0e470161c3c701d0649fe5873a27f6a87b365e',
-  })
-
+  source_url "https://download.gnome.org/sources/gtk+/3.24/gtk+-#{@_ver}.tar.xz"
+  source_sha256 `curl -Ls https://download.gnome.org/sources/gtk+/3.24/gtk+-#{@_ver}.sha256sum |\
+                 tail -n1 | cut -d ' ' -f1`.tr("\n", '') if (ARGV[0] == 'install' or ARGV[0] == 'reinstall')
```

Comment:
Would `.chomp` work better than `.tr("\n", '')`?  I'm thinking we should be checking for `-s`, `-S`, `--build-from-source` or `--recursive-build` switch options instead because technically `upgrade` could be a crew argument also.

Reply:
Actually, I don't think we need to check for any arguments.  Just get rid of `if (ARGV[0] == 'install' or ARGV[0] == 'reinstall')` altogether and it should still work fine.